internal/store/store.go

```diff
@@ -645,8 +656,8 @@ func (s *Store) PushCompose(composeID uuid.UUID, bp *blueprint.Blueprint, packag
 
 // PushComposeRequest is an alternative to PushCompose which does not assume a pre-defined distro, as such it is better
 // suited for RCM API and possible future API that would respect the fact that we can build any distro and any arch
-func (s *Store) PushComposeRequest(request common.ComposeRequest) error {
-	// This should never happen and once distro.Manifest is refactored this check will go away
+func (s *Store) PushComposeRequest(request ComposeRequest) error {
+	// This should never happen and once distro.Pipeline is refactored this check will go away
```

Comment:
Probably a left-over? `distro.Pipeline` is no longer a thing.

Reply:
Yeah, I wrote the patch before the pipeline → manifest switchover and messed up the rebase. Thanks for the catch! Fixed.